controllers/util.go

```diff
@@ -334,3 +334,12 @@ func getInvalidSmsReceivers(smsForm SmsForm) []string {
 	}
 	return invalidReceivers
 }
+
+func removeElementFromSlice[T comparable](slice []T, element T) []T {
```

Comment:
Please add tests for this.

Reply:
we already have DeleteVal func in util package, mb use it instead new func?